docfiles/docs.js

```diff
@@ -259,6 +261,35 @@ function renderSnippets() {
     });
 }
 
+function printDocument(type) {
+    let stylesSheet = [].slice.call(document.styleSheets).filter(function(x) { return String(x.href).indexOf("semantic") !== -1; })[0];
+    if (type === 'color') {
+        let rules = [].slice.call(stylesSheet.cssRules).filter(function(x,i) { 
+            if (String(x.media).indexOf("only print") !== -1) {
+                return x;
+        }});
+
+        window.printStyles = rules;
+
+        window.printStyles.forEach(function(x) {
+            let index = [].slice.call(stylesSheet.cssRules).indexOf(x);
+            stylesSheet.deleteRule(index);
```

Comment:
This is really hacky. I think we should add a “print-color” class on the body and have CSS rules get cancelled because of that.

Reply:
Since the default print mode is black & white, we loose the origin colors. We could try removing the current print.less classes and then have a new "black & white" mode on body as you say. I didn't want to move anything in print.less because I don't know how this will behave in lobby and editor.

Reply:
I changed the approach to use add/remove css clases from classList